wiki/People/The_Team/Beatmap_Nominators/General_Information/id.md

```diff
@@ -1,55 +1,56 @@
+- [Ranking Criteria](/wiki/Ranking_Criteria): Jika kamu sudah menjadi seorang BN, kamu harus terbiasa dengan ini. Pada dasarnya, ini berfungsi untuk menjelaskan aspek mapping yang diperbolehkan dan tidak diperbolehkan di bagian Ranked, serta beberapa pedoman umum yang harus diikuti dalam kebanyakan kasus.
+- [Informasi Beatmap Veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto): Untuk beatmap dengan masalah kualitas secara objektif yang sebetulnya tidak melanggar aturan dan pedoman dari Kriteria Ranking, tetapi yang menurut kamu membuat beatmap tersebut tidak sesuai untuk bagian Ranked, beatmap veto memungkinkan kamu menahan beatmap dari Kualifikasi.
 - [Mapset Verifier](https://osu.ppy.sh/community/forums/topics/943895): Sebuah alat yang dibuat oleh anggota NAT yaitu [Naxess](https://osu.ppy.sh/users/8129817). Alat ini memeriksa berbagai masalah unrankable, serta memiliki fitur snapshot agar lebih mudah dalam membandingkan versi map yang bermanfaat saat pengecekan ulang. **Ini tidak menggantikan pemeriksaan beatmap sendiri**, tetapi digunakan sebagai alat untuk membantu memeriksa ulang sehingga tidak ada yang terlewat.
 
 ## Kemampuan Beatmap Nominator
 
-Nominasi dan mereset nominasi adalah bukti dari usaha seorang Beatmap Nominator saat memeriksa map, menunjukkan kemajuan sebuah map melalui proses ranking. Sebagai seorang Beatmap Nominator kamu memiliki akses ke alat-alat ini dan harus menggunakannya dengan bijak.
+Nominasi dan mereset nominasi adalah bukti dari usaha seorang BN saat memeriksa map, menunjukkan kemajuan sebuah map melalui proses ranking. Sebagai seorang BN kamu memiliki akses ke alat-alat ini dan harus menggunakannya dengan bijak.
 
 ### Nominasi
 
 ![tombol nominasi](img/nominate_id.png)
 
-**Tombol ini memungkinkan kamu untuk menominasi beatmap.** Untuk menominasi sebuah beatmap, mapset harus memiliki setidaknya 5 Hype dan tidak ada tanda Masalah atau Saran yang saat ini belum terselesaikan. Pastikan kamu puas dengan semua tingkat kesulitan, bahkan mode permainan lainnya, sebelum menempatkan ini. Jika beatmapset sebelumnya diveto, kamu harus memastikan veto tersebut sudah diselesaikan sebelum memajukan mapset. Ketika sebuah beatmap memiliki 2 nominasi, itu akan dikualifikasi dan masuk ke antrian ranking.
+**Tombol ini memungkinkan kamu untuk menominasi beatmap.** Untuk menominasi sebuah beatmap, mapset harus memiliki setidaknya 5 Hype dan tidak ada tanda Masalah atau Saran yang saat ini belum terselesaikan. Pastikan kamu puas dengan semua tingkat kesulitan, bahkan mode permainan lainnya, sebelum menempatkan ini. Jika beatmap sebelumnya diveto, kamu harus memastikan veto tersebut sudah diselesaikan sebelum memajukan mapset. Ketika sebuah beatmap memiliki 2 nominasi, itu akan dikualifikasi dan masuk ke antrian ranking.
 
 ### Diskualifikasi atau mereset nominasi
 
 ![tombol diskualifikasi dan reset](img/reset_dq_id.png)
```

Comment:
```suggestion
![tombol diskualifikasi dan tombol reset](img/reset_dq_id.png)
```

Reply:
could it get the own word since it's doesn't have to attach with `diskualifikasi `